licenses.d2iq.yaml

```diff
@@ -595,12 +595,12 @@ resources:
       - url: https://github.com/prometheus-community/postgres_exporter
         ref: v0.12.0
         license_path: LICENSE
-  - container_image: ghcr.io/mesosphere/dkp-container-images/docker.io/nginxinc/nginx-unprivileged:1.25.5-alpine-d2iq.0
+  - container_image: ghcr.io/mesosphere/dkp-container-images/docker.io/nginxinc/nginx-unprivileged:1.27.1-alpine-d2iq.0
```

Comment:
change the image 
docker.io/nginxinc/nginx-unprivileged:1.27.1-alpine

Reply:
done